src/compiler/crystal/tools/doc/method.cr

```diff
@@ -175,4 +176,15 @@ class Crystal::Doc::Method
   def has_args?
     !@def.args.empty? || @def.block_arg || @def.yields
   end
+
+  def to_json(io)
+    {
+      name:     name,
+      args:     args_to_s,
+      doc:      doc,
+      prefix:   prefix,
```

Comment:
Currently all instance and class methods are listed in an array of the parent object, so for now I think I'll just remove the `kind` and `prefix` fields from method objects.